godot-cell/src/borrow_state.rs

```diff
@@ -128,6 +143,14 @@ impl BorrowState {
             .checked_add(1)
             .ok_or("could not increment shared count")?;
 
+        #[cfg(feature = "threads")]
+        self.thread_shared_count
+            .entry(std::thread::current().id())
+            .and_modify(|count| {
+                *count = count.saturating_add(1);
+            })
```

Comment:
Also here -- saturation is a non-issue in practice, but if we already don't use `+= 1`, then let's at least use `checked_add` so that everything explodes in practice.

Imo this is not needed though, `+=` and `-=` are fine and more readable. Debug mode will still tell us if we have overflows.

Reply:
the "global" `shared_count` should already cover over and under flow, so I'm fine with changing this to `-=` and `+=`.